src/main/java/com/softserve/service/impl/SemesterServiceImpl.java

```diff
@@ -60,14 +64,14 @@ public Semester getById(Long id) {
      */
     @Override
     public List<Semester> getAll() {
-        log.info("In getAll()");
+        log.debug("In getAll()");
         List<Semester> semesters = semesterRepository.getAll();
-        for (Semester semester:semesters) {
+        for (Semester semester : semesters) {
```

Comment:
Update query for this method to fetch in one shot instead of using Hibernate.initialize like "JOIN FETCH"

Reply:
The Hibernate.initialize approach is applied throughout the project, example in LessonServiceImpl and ScheduleServiceImpl.
Maybe it's worth creating a separate task to fix the whole project?

Reply:
Done

Reply:
I still see hibernate initialize here :)